### PR TITLE
Remove nonfunctional enableSnap function, update shortcuts

### PR DESCRIPTION
**Pull request by jcp2236**
I've briefly tested the functionality of drawing lines/line snapping after removing enableSnap, and it seems to be behaving normally. See #520

### CODE REVIEW

**Comment by sisskind** [comment]
jenkins test this please

File: js/id/actions/move.js [hunk 1]
```diff
@@ -169,39 +169,39 @@ iD.actions.Move = function(moveIds, tryDelta, projection, cache) {
...
-            graph = graph.replace(way1);
-        }
-        if (!isEP2 && edge2.index !== way2.nodes.indexOf(vertex.id)) {
-            way2 = way2.removeNode(vertex.id).addNode(vertex.id, edge2.index);
-            graph = graph.replace(way2);
-        }
-
-        return graph;
-    }
+    //     var edge1 = !isEP1 && iD.geo.chooseEdge(nodes1, projection(vertex.loc), projection),
```
Comment (sisskind) [inline js/id/actions/move.js]: @jcp2236 why comment this and not remove, as you did with the other code?

Comment (jcp2236) [inline js/id/actions/move.js]: Thanks for catching that.

**Comment by jcp2236** [comment]
jenkins run cucumber

**Pull request merged**